fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -2119,6 +2137,19 @@ public Builder setProgressLogIntervalMillis(long millis) {
             return this;
         }
 
+        /**
+         * Set whether or not to track the index build progress by updating the number of records successfully scanned
+         * and processed. The progress is persisted in {@link #indexBuildScannedRecordsSubspace(FDBRecordStore, Index)}
+         * which can be accessed by {@link IndexBuildState#getIndexBuildStateAsync(FDBRecordStore, Index)}, comparing
+         * to the records scanned information in progress log which cannot be accessed programmatically.
```

Comment:
I'm not sure what the last sentence here is trying to say. That the object allows one to compare the records scanned with the records in the record store? I'm not sure where the "programmatically" bit comes in.